OSDOCS-17319 updates to ESO documentation (#102710)
openshift/openshift-docs: modules/external-secrets-operator-install-console.adoc

Author: Done

[id="external-secrets-operator-install-console_{context}"]

> = Installing the {external-secrets-operator} by using the web console

Reviewer: The assembly for Installing the {external-secrets-operator} has an issue in the preview where the attribute does not render:

<img width="2508" height="268" alt="Screenshot From 2026-01-06 14-19-40" src="https://github.com/user-attachments/assets/9ca6a821-fe98-49da-87f5-16b3bc1061f2" />

Correct attribute?

[role="_abstract"]

> Install the {external-secrets-operator} using the web console to add secret management features to your cluster. This process lets you select an update channel and approval strategy to help ensure the Operator stays current.

Reviewer: ```suggestion
Install the {external-secrets-operator} by using the web console to add secret management features to your cluster. By doing this task, you can select an update channel and approval strategy to help ensure the Operator stays current.
```

Author: Done
